### PR TITLE
Update Create a UP guide-low level

### PR DESCRIPTION
**Pull request by biancabuzea200**
-updated guide Create a UP(low level) to reflect the L16->Testnet change

### CODE REVIEW

File: docs/guides/universal-profile/create-profile.md [hunk 1]
```diff
@@ -308,22 +308,22 @@ async function createUniversalProfile() {
 
 createUniversalProfile();
 
-// my Universal Profile address: 0xEde1198b046d8ED64B71adeA5d3B7370cc84A7FB
+// my Universal Profile address: 0xbd9Ef16132258ACbfEd899F594C45772604C6509
 ```
 
-We can also visualize our UP on the [l16.universalprofile.cloud](https://l16.universalprofile.cloud) website by adding the address of the deployed UP in the URL, after the `/` (slash), as follow:
+We can also visualize our UP on the [universalprofile.cloud](https://universalprofile.cloud) website by adding the address of the deployed UP in the URL, after the `/` (slash), as follow:
```
Comment (Hugoo) [inline docs/guides/universal-profile/create-profile.md:314]: As of right now, this is not correct.

File: docs/guides/universal-profile/create-profile.md [hunk 2]
```diff
@@ -308,22 +308,22 @@ async function createUniversalProfile() {
...
 
-*https://l16.universalprofile.cloud/{your-up-address}*
+*https://universalprofile.cloud/{your-up-address}*
```
Comment (Hugoo) [inline docs/guides/universal-profile/create-profile.md:316]: Not true

**Review by Hugoo — CHANGES_REQUESTED** [review]

File: docs/guides/universal-profile/create-profile.md (hunk 2 above)
Comment (biancabuzea200) [reply docs/guides/universal-profile/create-profile.md]: replaced it with wallet.universalprofile.cloud

File: docs/guides/universal-profile/create-profile.md (hunk 1 above)
Comment (biancabuzea200) [reply docs/guides/universal-profile/create-profile.md]: replaced it with wallet.universalprofile.cloud and enforced smart contract version

File: docs/guides/universal-profile/create-profile.md [hunk 3]
```diff
@@ -5,16 +5,23 @@ sidebar_position: 1
...
 A complete _"ready to use"_ JS file is available at the end in the [**Final Code**](#final-code) section. If you want to run the code as standalone JavaScript files within the terminal or the browser, you can open the [`lukso-playground`](https://github.com/lukso-network/lukso-playground) repository or use the correlated [StackBlitz](https://stackblitz.com/github/lukso-network/lukso-playground) page.
 :::
 
 In this guide, we will learn how to:
 
 - create a Universal Profile.
-- see our new Universal Profile on [l16.universalprofile.cloud](https://l16.universalprofile.cloud).
+- see our new Universal Profile on [wallet.universalprofile.cloud](https://wallet.universalprofile.cloud/).
 
-![Universal Profile example on universalprofile.cloud](./img/my-up.png)
```
[the remark this answers is not in the document]
Comment (Hugoo) [reply docs/guides/universal-profile/create-profile.md:17]: I did the change in the last commit

**Review by Hugoo — APPROVED** [review]

**Pull request merged**